### PR TITLE
Load list column definitions on startup

### PR DESCRIPTION
**Pull request by ivo-ritense**

### CODE REVIEW

File: case/src/main/kotlin/com/ritense/case/service/CaseListDeploymentService.kt [hunk 1]
```diff
@@ -0,0 +1,98 @@
+/*
+ * Copyright 2015-2022 Ritense BV, the Netherlands.
```
Comment (marijnritense) [inline case/src/main/kotlin/com/ritense/case/service/CaseListDeploymentService.kt:2]: 2023?

File: case/src/main/kotlin/com/ritense/case/service/CaseListDeploymentService.kt [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
+        return resourcePatternResolver.getResources(CASE_LIST_DEFINITIONS_PATH)
+    }
+
+    private fun loadCaseListSchemaResource(): Resource {
+        return resourcePatternResolver.getResource(CASE_LIST_SCHEMA_PATH)
+    }
+
+    companion object {
+        internal const val CASE_LIST_SCHEMA_PATH = "classpath:config/case/schema/case-list.schema.json"
+        internal const val CASE_LIST_DEFINITIONS_PATH = "classpath:config/case/list/*.json"
```
Comment (marijnritense) [inline case/src/main/kotlin/com/ritense/case/service/CaseListDeploymentService.kt:95]: I forget, didn't we talk about wanting to allow implementation to separate case definitions as well?

**Review by marijnritense — APPROVED** [review]

File: case/src/main/resources/config/case/schema/case-list.schema.json [hunk 3]
```diff
@@ -0,0 +1,51 @@
+{
+    "$schema": "http://json-schema.org/draft-07/schema#",
+    "$id": "http://json-schema.org/draft-07/schema#",
+    "title": "Form flow definition",
+    "description": "Representation of a form flow definition",
```
Comment (ThomasRitense) [inline case/src/main/resources/config/case/schema/case-list.schema.json:5]: Representation of form flow?

Comment (luis-soares-ritense) [reply case/src/main/resources/config/case/schema/case-list.schema.json]: fixed in new commit

**Review by luis-soares-ritense — APPROVED** [review]

**Pull request merged**